05_syntax_lab/03.py

```diff
@@ -4,4 +4,14 @@
 For example if the number was: 2345
 The result should be: 14
 """
+# Exercise 3
+# By Zvi Figov
+from random import randint
+
+num = randint(1,10000)
+numstr = str(num)
+acc = 0
+for n in range(0,len(numstr)):
+    acc = acc + int(numstr[n])
+print "The sum of the digits of the number %d is %d" % (num,acc)
 
```

Comment:
You can iterate over the characters of a string using ```for c in numstr:``` that will further simplify the code.